test/python/test_temporal_reduction.py

```diff
@@ -16,7 +16,7 @@
 argc = len(sys.argv)
 if argc < 8:
     sys.stderr.write('usage: app [in file regex] [out file base] '
-                     '[n threads] [steps per file] [interval] [operator] '
+                     '[steps per file] [n threads] [interval] [operator] '
```

Comment:
why is this needed?

Reply:
They are switched in the code where `steps_per_file = int(sys.argv[3])` & `n_threads = int(sys.argv[4])`

Reply:
oops, I see it now.